utils/upgrade

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/python3
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import logging
+import sys
+import argparse
+import subprocess
+import os
+from pathlib import Path
+import re
```

Comment:
looks like following imports are not necessary: logging, subprocess, os, re (probably importing * from upgrade_utils makes them redundant)

Reply:
Changed it to not use wildcard import